package.json

```diff
@@ -16,12 +16,11 @@
   "scripts": {
     "prettify": "prettier --write lib/**/*.css && balena-lint --fix --typescript typings lib tests forge.config.ts forge.sidecar.ts webpack.config.ts",
     "lint": "npm run prettify && catch-uncommitted",
-    "test-gui": "xvfb-maybe electron-mocha --recursive --reporter spec --window-config tests/gui/window-config.json --require ts-node/register/transpile-only --require-main tests/gui/allow-renderer-process-reuse.ts --full-trace --no-sandbox --renderer tests/gui/**/*.ts",
-    "test-shared": "xvfb-maybe electron-mocha --recursive --reporter spec --require ts-node/register/transpile-only --require-main tests/gui/allow-renderer-process-reuse.ts --full-trace --no-sandbox tests/shared/**/*.ts",
-    "test": "npm run test-gui && npm run test-shared",
+    "test": "echo 'Only use custom tests; if you want to test locally, use `npm run wdio`' && exit 0",
```

Comment:
I don’t understand why you needed to make the test script a noop? From what I see the wdio command could just as well run in the test script. Can you elaborate?

Reply:
There's no point at running tests in against node as this is not a valid target. 
For the wdio test to run, we need a valid build. So it would be `npm run make && npm run wdio`. Which would be just a duplicate of the custom tests.